zha/application/platforms/light/__init__.py

```diff
@@ -540,6 +544,10 @@ async def async_handle_color_commands(
     ):
         """Process ZCL color commands."""
 
+        if self._color_cluster_handler is None:
+            self.debug("no color cluster handler available")
```

Comment:
We can probably change this to `warning` as well, similar to the warning for the fan cluster handler missing.
Realistically, we should never hit this if the light doesn't support colors/has a `Color` cluster (handler). If we do, we want a warning IMO.